packages/x-date-pickers/src/MonthCalendar/PickersMonth.tsx

```diff
@@ -1,16 +1,19 @@
 import * as React from 'react';
 import clsx from 'clsx';
-import { styled, alpha, useThemeProps } from '@mui/material/styles';
+import { alpha } from '@mui/material/styles';
```

Comment:
Will keeping the `alpha` call outside of `zero-styled` allow for static extraction? 🤔

Reply:
I think so, but @brijeshb42 can confirm/infirm this one :+1: